db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Card.create(original_text:"Fine", transcription: "[faɪn]",translated_text: "Сухой (о погоде), хороший, штраф", review_date: '10-10-2015')
```

Comment:
Line is too long. [137/80]<br>Space missing after colon.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.